operating/examine-images-for-vulnerabilities.adoc

```diff
@@ -59,14 +59,44 @@ It might fail to identify installed components or vulnerabilities if the final i
 
 |===
 
+//About Scanner V4
 include::modules/consolidated-scanner.adoc[leveloffset=+1]
 
+[id="enabling_scanner_v4_{context}"]
+== Enabling Scanner V4
+
+Scanner V4 is not enabled by default, but you can enable Scanner V4 during or after installation. Scanner V4 also requires that the StackRox Scanner remain enabled. Do not disable the StackRox Scanner even if Scanner V4 is enabled.
```

Comment:
> Scanner V4 also requires that the StackRox Scanner remain enabled.

This isn't *exactly* true. Users may remove StackRox Scanner, but they'll lose out on *KS Platform vulnerabilities as well as non-RHCOS node vulnerabilities. Perhaps we can make that clear that we recommend they keep it on so they do not miss out on those?

Reply:
Ah, OK, I thought that there was another reason that it had to be enabled, but I will clarify.

Reply:
We sometimes use "recommend" but we aren't really supposed to: 

```
Avoid "recommends". Instead of "Red Hat recommends", direct users to take the recommended action. This allows Red Hat to be more prescriptive in documentation and prevent any user uncertainty, and is easier for upstream or downstream coordinated efforts.

For example, instead of "Red Hat recommends using X package because", write "Use this package because" or "Use this package when".
```
I'll rewrite to try to explain why they should keep it enabled.